project/plugins.sbt

```diff
@@ -6,7 +6,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.15")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.18")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
```

Comment:
Missing plugin for `dependencyUpdates` (the command fails at the moment when running `run_all_tests.sh`):
`addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.3")`